pysurfex/pseudoobs.py

```diff
@@ -56,7 +66,21 @@ def snow_pseudo_obs_cryoclim(
     iii = 0
     res_lons = []
     res_lats = []
+
+    if neighbourhood:
+        cryo_radius = 2
```

Comment:
For later some of these settings could enter a config?

Reply:
Agree